packages/css/src/components/Button.css

```diff
@@ -44,78 +37,55 @@
   }
 }
 
-/* Primary intent */
-
-.Button.is-primary {
-  background-image: var(--gradient-rr);
-  background-color: var(--color-rasberry);
-  color: var(--color-white);
-  box-shadow: inset 0 -2px 0 0 var(--color-ruby);
-}
-
-.Button.is-primary:hover,
-.Button.is-primary.is-hover,
-.Button.is-primary:active,
-.Button.is-primary.is-active,
-.Button.is-primary[disabled],
-.Button.is-primary.is-disabled {
-  background-image: initial;
-  box-shadow: initial;
-}
-
-.Button.is-primary:active,
-.Button.is-primary.is-active {
-  background-color: var(--color-ruby);
+.Button[disabled],
+.Button.is-disabled {
+  background-color: var(--color-ui-element-primary) !important;
+  color: var(--color-ui-text-tertiary) !important;
+  cursor: not-allowed !important;
```

Comment:
Those rules were overridden later because of the specificity.
But I added `!important` here because no matter what, we want disabled buttons to have this style